src/averageAges.js

```diff
@@ -55,7 +71,18 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  function personMother(person) {
```

Comment:
```suggestion
  function findMother(person) {
```
Functions should be named with imperative verb and answer to "What to do?"